spec.md

```diff
@@ -2703,7 +2782,7 @@ info = "@" , "[" , { string , " " , linecol } , "]" ;
 (* Type definitions *)
 width = "<" , int , ">" ;
 binarypoint = "<<" , int , ">>" ;
-type_ground = "Clock"
+type_ground = "Clock" | "AsyncReset"
```

Comment:
Where is Reset (uninferred) in this table?

Reply:
Also we need to be careful if we just stick `Reset` in here, because this is used in extmodule.port.type ... is it legal to have a `Reset` here for extmodule? I'm not sure because Chisel doesn't even let you generate that.